src/components/tableDrawing/tableDrawing.module.scss

```diff
@@ -16,15 +16,15 @@
         align-items: center;
         display: flex;
         align-items: center;
+        border-top: 1px solid #000;
+        border-bottom: 1px solid #000;
     }
 
 
     .add_property_btn {
         height: 30px;
         background-color: #fff;
```

Comment:
It's good practice to keep repeating styles and colors of similar elements in a variable.
So that if we want to change it, we change it from one place.
Now it is not necessary to do a refactor, but in the future I will introduce it.

Reply:
I will try to make it more compact